.travis.yml

```diff
@@ -0,0 +1,27 @@
+---
+language: python
+python: "2.7"
+
+env:
+  - PLAYBOOK=test.yml
+
+install:
+  # Install Ansible.
+  - pip install ansible
+
+  # Add ansible.cfg to pick up roles path.
+  - "{ echo '[defaults]'; echo 'roles_path = ../'; } >> ansible.cfg"
+
+script:
+  # Check the role/playbook's syntax.
+  - "ansible-playbook -i tests/inventory tests/$PLAYBOOK --syntax-check"
+
+  # Run the role/playbook with ansible-playbook.
+  - "ansible-playbook -i tests/inventory tests/$PLAYBOOK --connection=local --sudo"
+
+  # Run the role/playbook again, checking to make sure it's idempotent.
```

Comment:
Later on I'd like to plug in TestKitchen and ServerSpec but for the time being this is better than nothing. I really want us to be able to provide not only a great service but also quality roles.